wallet/psbt_test.go

```diff
@@ -217,6 +218,146 @@ func TestFundPsbt(t *testing.T) {
 				"index after sorting",
 			)
 		},
+	}, {
+		name: "one input and a custom change scope: BIP0086",
+		packet: &psbt.Packet{
+			UnsignedTx: &wire.MsgTx{
+				TxIn: []*wire.TxIn{{
+					PreviousOutPoint: utxo1,
+				}},
+			},
+			Inputs: []psbt.PInput{{}},
+		},
+		feeRateSatPerKB:         20000,
+		validatePackage:         true,
+		changeKeyScope:          &waddrmgr.KeyScopeBIP0086,
+		expectedInputs:          []wire.OutPoint{utxo1},
+		expectedChangeBeforeFee: utxo1Amount,
+	}, {
+		name: "one input and a custom change scope: BIP0084",
+		packet: &psbt.Packet{
+			UnsignedTx: &wire.MsgTx{
+				TxIn: []*wire.TxIn{{
+					PreviousOutPoint: utxo1,
+				}},
+			},
+			Inputs: []psbt.PInput{{}},
+		},
+		feeRateSatPerKB:         20000,
+		validatePackage:         true,
+		changeKeyScope:          &waddrmgr.KeyScopeBIP0084,
```

Comment:
Is it worth testing all the different scopes because it seems that the relevant code paths are no different?

Reply:
Otherwise a parameterized test may be nice, but possibly overkill.

Reply:
No strong opinion on generic assert. I do think that it is nice that you also assert the default change scope.